src/database/migrations/1600909362948-CreateTeachers.ts

```diff
@@ -0,0 +1,54 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export default class CreateTeachers1600909362948 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Adicionar essa linha:
await queryRunner.query(`CREATE EXTENSION IF NOT EXISTS "uuid-ossp"`);

Oque essa linha faz? O banco precisar ter a extensão "uuid-ossp", quando estamos trabalhando com o método de geração default: uuid_generate_v4(). Então, para não precisarmos adicionar manualmente no banco, é bom colocar como a primeira query rodando.

Reply:
@AntonioNtV preciso adicionar essa query em todas as migrations que usam o uuid_generate_v4()? E eu preciso adicionar algo no down quanto à isso?

Reply:
@davigsousa essa query é só rodada na primeira  migration.
E  o drown, se não me engano, é 
 await queryRunner.query(DROP EXTENSION IF EXISTS "uuid-ossp");